app/controllers/CategoryGuidanceController.scala

```diff
@@ -40,20 +40,20 @@ class CategoryGuidanceController @Inject() (
   identify: IdentifierAction,
   getData: DataRetrievalAction,
   requireData: DataRequiredAction,
+  profileAuth: ProfileAuthenticateAction,
   val controllerComponents: MessagesControllerComponents,
   view: CategoryGuidanceView,
   auditService: AuditService,
   categorisationService: CategorisationService,
   navigator: Navigator,
-  goodsRecordConnector: GoodsRecordConnector,
-  sessionRepository: SessionRepository
+  goodsRecordConnector: GoodsRecordConnector
```

Comment:
can we also remove the import for SessionRepository

Reply:
Removed